inc/spbc-settings.php

```diff
@@ -2324,6 +2407,67 @@ function spbc_field_traffic_control_logs__prepare_data(&$table)
     }
 }
 
+/**
+ * Creates a banner with a notification
+ * @param string $flag_text_banner which banner text to show
+ * @return void
+ */
+function spbc_settings__create_notice_on_tab($flag_text_banner)
+{
+    global $spbc;
+
+    switch ($flag_text_banner) {
+        case 'found_critical_files': // Creates a banner with a notification if important files are found and if the banner has not already been closed
+            if (!array_key_exists('spbct_notice-found_critical_files', $_COOKIE)) {
+                $_COOKIE['spbct_notice-found_critical_files'] = '0';
```

Comment:
Please use a wrapper `Cookie::get();` instead of direct access to the `$_COOKIE` array